main.go

```diff
@@ -74,13 +86,24 @@ func main() {
 
 	loadedPlugins = plugins.GetPlugins(client)
 
-	log.Info("Lassie is now watching for some work!")
+	log.Info("Lassie is now waiting for some work!")
 
-	s := gocron.NewScheduler(time.UTC)
-	s.SetMaxConcurrentJobs(1, gocron.RescheduleMode) // prevent parallel execution and skip if last run hasn't finished yet
-	s.Every(30).Seconds().Do(func() {
-		loop(client)
+	updateInterval := 30
+	if os.Getenv("LOG") == "debug" {
+		updateInterval = 5
+	}
+
+	sC := gocron.NewScheduler(time.UTC)
+	sC.SetMaxConcurrentJobs(1, gocron.RescheduleMode)
+	sC.Every(5).Minutes().Do(func() {
```

Comment:
I'm not sure if it is a good idea to have different schedules for config reading and running plugins.

Reply:
@lukashass This seems to be the easiest way for me to do it for now.

Reply:
What about reading the config every time, because it might have changed. That would be even easier IMHO. Are you worried about performance?

Reply:
And I'm concerned about concurrent access on the config object. Not sure if that can cause a problem with two schedules?

Reply:
@lukashass I was trying that at the beginning, but it seemed to slow down the loop. I reverted it anyways to not need to lock anything. Hopefully works for now.